user-guide/Advanced_Functionality/Swarming/Swarming.md

```diff
@@ -0,0 +1,15 @@
+---
+uid: Swarming
+---
+
+# Swarming
+
+With the upcoming DataMiner Swarming functionality, you will be able to [switch objects](xref:SwitchingObjects), such as elements or services, from one DataMiner Agent to another Agent in that cluster.
+
+The primary goal of Swarming is to eliminate downtime resulting from maintenance activities and provide a more polished user experience.
+
+> [!NOTE]
+> Swarming is not available in DataMiner Systems with a [storage per DMA setup](xref:Configuring_storage_per_DMA) (Cassandra or MySQL).
```

Comment:
Leaving a disclaimer that mentions in what systems DataMiner Swarming is NOT available might be easier as long as STaaS hasn't been made available to the public yet.

Reply:
Good idea to mention the storage per DMA setup indeed